drivers/volume/volume.go

```diff
@@ -54,7 +54,7 @@ type Driver interface {
 
 	// ValidateCreateVolume validates whether a volume has been created properly.
 	// params are the custom volume options passed when creating the volume.
-	ValidateCreateVolume(name string, params map[string]string) error
```

Comment:
This PR has improved a lot.  I'm mostly OK with what is in it, but I am a bit hesitant about
this change to `ValidateCreateVolume()`.  Before your PR, this function was fairly simple in that it took in a name and params, and then validated if a volume was up/proper.

After your change you internally changed it to loop over many volumes.  This is a major semantic change to this function.

Is it possible for you to:

1.  Keep ValidateCreateVolume() the way it was before
2.  Create a new function which takes an array or map of volumes, and in turn calls ValidateCreateVolume() in a loop?

This would ease my mind, and help me push this PR into the tree.